srml/council/src/seats.rs

```diff
@@ -345,32 +351,34 @@ decl_storage! {
 		/// A list of votes for each voter, respecting the last cleared vote index that this voter was
 		/// last active at.
 		pub ApprovalsOf get(approvals_of): map T::AccountId => Vec<bool>;
-		/// The vote index and list slot that the candidate `who` was registered or `None` if they are not
-		/// currently registered.
+		/// The vote index and list slot in which the candidate account ID was registered or `None` if
+		/// they are not currently registered.
 		pub RegisterInfoOf get(candidate_reg_info): map T::AccountId => Option<(VoteIndex, u32)>;
 		/// The last cleared vote index that this voter was last active at.
 		pub LastActiveOf get(voter_last_active): map T::AccountId => Option<VoteIndex>;
 		/// The present voter list.
 		pub Voters get(voters): Vec<T::AccountId>;
 		/// The present candidate list.
 		pub Candidates get(candidates): Vec<T::AccountId>; // has holes
+		/// Number of candidates.
 		pub CandidateCount get(candidate_count): u32;
 
 		// temporary state (only relevant during finalization/presentation)
-		/// The accounts holding the seats that will become free on the next tally.
+		/// The accounts holding the seats that will become free on the next tally. Tuple of the block number
+		/// at which the next tally will occur, the number of seats, and a list of the account IDs.
 		pub NextFinalize get(next_finalize): Option<(T::BlockNumber, u32, Vec<T::AccountId>)>;
```

Comment:
could we declare a type called `pub type AvailableSeatsCount = u32;`

and then change the `u32` here to be `AvailableSeatsCount` so it's more self-describing

Reply:
@ltfschoen I agree with the general recommendation that maybe so many `u32`s being spread around makes the code unreadable and they should be aliased to a unified `pub type VoteCount = u32;` or something similar; but best if we continue on that in a separate PR?

Reply:
@Kianenigma yes absolutely, it was just an observation for a seperate PR